src/Symfony/Bridge/Monolog/Handler/ConsoleHandler.php

```diff
@@ -173,4 +194,9 @@ private function updateLevel(): bool
 
         return true;
     }
+
+    private function isInteractiveOnlyEnabled(): bool
+    {
+        return $this->interactiveOnly && $this->input && $this->input->isInteractive();
```

Comment:
Why do we need the `interactiveOnly` flag? Is determining if the console is interactive via the input not enough?